skrifa/src/outline/autohint/outline.rs

```diff
@@ -197,6 +202,35 @@ impl Outline {
         self.points.clear();
         self.contours.clear();
     }
+
+    pub fn to_path(
+        &self,
+        style: PathStyle,
+        pen: &mut impl OutlinePen,
+    ) -> Result<(), path::ToPathError> {
+        for contour in &self.contours {
+            let Some(points) = self.points.get(contour.range()) else {
+                continue;
+            };
+            #[inline(always)]
+            fn to_contour_point(point: &Point) -> path::ContourPoint<F26Dot6> {
```

Comment:
If you hoist this out to a free fn it would be *much* easier for me to read, inline it obfuscates the control flow.

Reply:
Made this a method on `Point`